.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    name: Node ${{ matrix.node }} / ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest]
```

Comment:
parcelを用いたビルドが最新のubuntuでは動かないので、window環境のみでtestやreleaseを行うようにしています。
下のようなエラーメッセージの通りちょいちょい動かない環境があるらしいです。
> No prebuild or local build of @parcel/watcher found. Tried @parcel/watcher-linux-x64-glibc. Please ensure it is installed (don't use --no-optional when installing with npm). Otherwise it is possible we don't support your platform yet. If this is the case, please report an issue to https://github.com/parcel-bundler/watcher.

Reply:
議論:
- @ShinobuTakahashi さん曰く「同じ問題を再現したが package-lock を削除したら回避できた」
   - 恐らく Parcel は CPU アーキテクチャごとに package を分けていて、package-lock.json が環境依存になってしまう設計になっている＝この問題は不可避
   - `--no-package-lock` で回避できなくはないが、CI で好き好んでやることではなさそう
- 短期的には Windows のみで差し支えないが、将来的には次のいずれかを考えるべきか
   - A) 脱 Parcel
   - B) テスト用のビルドプロセスから gamejson-helper を省く
      - Parcel を使っているのは gamejson-helper で、ランタイムとは独立している
      - ただし究極的には gamejson-helper もテストがあるべきで、理想解ではない

Reply:
ありがとうございます！将来的な対応についてはAとBどちらを選択するかも含めて別で対応しようと思います。